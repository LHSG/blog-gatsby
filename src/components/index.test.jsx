import * as index from '.';

test('exports', () => {
  expect(index).toMatchSnapshot();
});
