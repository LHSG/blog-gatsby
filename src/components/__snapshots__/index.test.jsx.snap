// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports 1`] = `
Object {
  "About": [Function],
  "Head": [Function],
  "IndexPage": [Function],
  "Layout": [Function],
  "Post": [Function],
}
`;
